solution/build.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko
+PRODUCT_LOWERNAME=zenko
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=unix:///var/run/docker.sock
+HARDLINK=hardlink
+OPERATOR_SDK=operator-sdk
+OPERATOR_SDK_OPTS=
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+
+# grab our dependencies from our deps.txt file as an array
+readarray -t DEP_IMAGES < deps.txt
+# add the CR dependencies by joining image: and tag: lines
+readarray -t CR_IMAGES < <(grep image: zenko-version-cr.yaml | awk '{print $2}')
```

Comment:
It might be the best option for now to `grep` in the yaml files, but it would be more robust if some sort of `jq` for yaml could be used instead.

Reply:
Does jq work for yaml or are you saying if something similar did exist? Cause that would be swell.

Reply:
I don't know if a `jq` for yaml exists, but it would be useful if it does! Otherwise I'm sure some python scripting can do the job too.

Reply:
A middle ground would be to write the `zenko-version-cr.yaml` file in json instead and use `jq`, as the kube client tools don't really care if it's json or yaml in general (we humans just tend to do yaml more).

Reply:
The big reason I am using the yaml in the first place was to copy paste it from zenko-operator. If I need to rewrite it I think just merging it into the deps.txt would be fine.

Reply:
@RachedBenMustapha I merged the zenko-version-cr.yaml into deps for dependencies and removed the paths from the images in the yaml as, from what I can tell, that is what is needed when using the private registry. Although, really just guessing a little here.

Reply:
(JSON is valid YAML :wink:)